server.js

```diff
@@ -48,9 +56,9 @@ app.post('/api/v1/pets', (request, response) => {
     }
   }
 
-  const { name, type } = pet;
-  app.locals.pets.push({ name, type, id });
-  response.status(201).json({ name, type, id });
+  const { petName, petNickname, petAge, petFunFact, petOwnersName} = pet;
```

Comment:
Naming things can be hard! Is it necessary to include the word `pet` in all these property values, or could you get away with `name, nickname, age, funFact, ownersName` - the inclusion of `pet` could be seen as duplicated code/naming